tests/fuzzing/oss_fuzz_build.sh

```diff
@@ -4,10 +4,10 @@
 # if run outside of it.
 # The api, compile_go_fuzzer() is provided by the OSS-fuzz
 # environment and is a high level helper function for a series
-# of compilation and linking steps to build the fuzzers in the 
+# of compilation and linking steps to build the fuzzers in the
 # OSS-fuzz environment.
 # More info about compile_go_fuzzer() can be found here:
 #     https://google.github.io/oss-fuzz/getting-started/new-project-guide/go-lang/#buildsh
-compile_go_fuzzer ./libcontainer/system FuzzUIDMap id_map_fuzzer linux
-compile_go_fuzzer ./libcontainer/user FuzzUser user_fuzzer
-compile_go_fuzzer ./libcontainer/configs FuzzUnmarshalJSON configs_fuzzer
+compile_go_fuzzer github.com/opencontainers/runc/libcontainer/system FuzzUIDMap id_map_fuzzer linux,gofuzz
```

Comment:
The `linux` tag is probably redundant -- as long as fuzzing is done on Linux.

Reply:
> OSS-fuzz only fuzz on Linux.

This means that the linux tag is there implicitly (set by the platform), and so it is redundant.

Reply:
Anyway, this can be addressed separately